module.exports = async(error) => {
    console.log('Aconteceu um erro :')
	console.log(error);
}